__tests__/mindfulness.test.tsx

```diff
@@ -6,11 +6,34 @@ jest.mock('next/navigation', () => jest.requireActual('next-router-mock'))
 jest.mock('../lib/zoomapi', () => jest.requireActual('../lib/fakezoomapi'));
 
 describe('Page', () => {
-  it('renders a heading', () => {
+  it('renders a heading and two videos', () => {
     render(<Page />)
 
     const heading = screen.getByRole('heading', { level: 1 })
 
     expect(heading).toBeInTheDocument()
+
+    const video1 = screen.getByTestId('youtube1-iframe');
```

Comment:
Rather then getting the videos by testid, let's query `ByTitle`

It looks like the title attribute should be a descriptive summary of what's in the video rather than just the generic "Youtube player".

Check out this blog post if you'd like to learn more: https://medium.com/web-dev-survey-from-kyoto/a-quick-note-on-embedding-a-youtube-video-in-2021-4c7ce731bfc4

Reply:
solved. query by title did not work (at least for me), I used aria-label